Sileo/Sileo/nb.lproj/Localizable.strings

```diff
@@ -0,0 +1,158 @@
+/* 
+  Localizable.strings
+  Sileo
+
+  Created by AppleBetas on 2018-12-27.
+  Translated to Norwegian Bokmål by : WorldlyDev (2020-04-21).
+  Copyright © 2018 CoolStar. All rights reserved.
+ 
+ NORWEGIAN BOKMÅL LOCALISATIONS
+*/
+
+// Credit Information (optional, uncomment to specify)
+//"__LocalizationCredit.Title" = "Sileo oversatt til Norsk Bokmål av WorldlyDev";
+//"__LocalizationCredit.URL" = "https://twitter.com/worldlydev"; // (optional)
```

Comment:
Hi, thanks for submitting translations! Have you tested these with the current (1.7.4 I think) release of Sileo?

Reply:
Just fixed this.

I haven't been able to test this as I'm currently on Unc0ver (11 pro, 13.3) and don't have Sileo as my default. If you give me a couple of minutes I will be able to see if it all looks good.

Was the Localizable.stringsdict done correctly? Not really sure what it is to be honest. This kind of coding goes way above my level of understanting :)